adaptivefiltering/utils.py

```diff
@@ -1,2 +1,12 @@
+import collections
+
+
 class AdaptiveFilteringError(Exception):
     pass
+
+
+def is_iterable(object):
+    if isinstance(object, collections.abc.Iterable) and not isinstance(object, str):
+        return True
+    else:
+        return False
```

Comment:
This is equivalent to `return isinstance(object, collections.abc.Iterable) and not isinstance(object, str)`